server/real_estate/src/main/java/com/realestate/real_estate/controllers/BrokersController.java

```diff
@@ -16,8 +17,9 @@
 public class BrokersController {
     @Autowired
     private BrokerService brokerService;
+    
     @GetMapping(path = "/properties")
-    public ResponseEntity<List<Property>> getAllProperties() {
+    public ResponseEntity<List<PropertyDetails>> getAllProperties() {
```

Comment:
This is one of my endpoint right? It's supposed to return List<Property> not List<PropertyDetail>

Reply:
Fixed it